proto/api.proto

```diff
@@ -840,6 +851,15 @@ enum MarketStatus {
   MS_ONLINE = 1; // online
 }
 
+message PostSubmitMineOreResponse {
+  string status = 1;
+}
+
+message PostSubmitMineOreRequest {
+  PostSubmitRequest first_transaction = 1;
```

Comment:
no need to have 2 dedicated fields with complex format...just do:
` repeated string transactions = 1;`